DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -3,7 +3,7 @@
 
 // -*- C++ -*-
 //
-// Package:    DetectorDescription/DDFilteredView
+// Package:    DetectorDescription/Core
```

Comment:
I don't understand. This file is not in Core.

Reply:
no, but it should be

Reply:
Is the idea that the DDCMS package will be moved to Core once the migration is complete?

Reply:
yes, DDCMS package was a temporary placeholder for all DD4hep related development.